modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1586,19 +1586,52 @@ private ObjectEntry _toObjectEntry(
 						objectDefinition.getObjectDefinitionId(),
 						ObjectActionTriggerConstants.KEY_STANDALONE)) {
 
+				HashMap<String, String> templateParameterMap =
+					HashMapBuilder.put(
+						"objectActionName", objectAction.getName()
+					).put(
+						"scopeKey",
+						() -> {
+							if (Objects.equals(
+									objectDefinition.getScope(), "site")) {
+
+								return String.valueOf(
+									serviceBuilderObjectEntry.getGroupId());
+							}
+
+							return null;
+						}
+					).build();
+
+				String methodName = null;
+
+				String serviceBuilderObjectEntryExternalReferenceCode =
+					serviceBuilderObjectEntry.getExternalReferenceCode();
+
+				if (Objects.equals(objectDefinition.getScope(), "site")) {
```

Comment:
As this is always the same value for each iteration of the loop, you could calculate if before starting the loop. Something like:

```java
boolean scopeSite = Objects.equals(
	objectDefinition.getScope(),
	ObjectDefinitionConstants.SCOPE_SITE);
```